support/ml-advice/index.rst

```diff
@@ -14,8 +14,187 @@
 .. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link and explicit title using the syntax :ref:`link text <label-name>`.
 .. A warning will alert you of identical labels during the linkcheck process.
 
-########################################
-Advice for machine learning integrations
-########################################
+######################################################
+Considerations for integrating AI with Citizen Science
+######################################################
 
-*Advice, resources, and best practices for the integration of machine learning techniques into Citizen Science projects to be added here.*
+
+Overview
+========
+
+.. Purpose
+The goal of this page is to provide background, references, and advice for Rubin users who would like to use artificial intelligence (AI) -- otherwise known as machine learning (ML) -- within or alongside their citizen science (CS) research workflows. 
+AI and CS are overlapping and complementary procedures for rapidly assessing (typically, classifying) scientific data. 
+They have similar and unique pros and cons, and there are multiple ways to use these two methods in concert to improve the quality of your scientific results. 
+In addition to this advice, there are tutorials for doing citizen science tutorials.
+It is recommended that you study basic AI principles and methods, including performing your own small computational studies, before embarking on a CS project that integrates AI. 
+
+.. Context for the rest
+Almost all of the discussion on this page is in terms of classification tasks; 
+it will consider the specific task of classifying 'stars' and 'galaxies' as an example when needed. 
+Additionally, this discussion will focus on classifying images, because this is the most common application of both AI and CS. 
+
+
+
+
+Comparing AI and CS
+===================
+
+AI and CS have multiple similarities and differences. 
+Identifying these items for your project is important for determining the best use of AI in your Rubin CS project. 
+
+
+Similarities
+------------
+
+The similarities between AI and CS include the following:
+
+- Both methods take large sets of input data and output a classification score for each object in the set. The output scores are not inherently probabilistic, but they can be calibrated to be nearly probabilistic.
+- To an approximation, they can both be treated as ''black boxes'' because it remains an active area of research to interpret the ''reasons'' that an algorithm or a human would classify an object. Using statistical metrics helps in both cases, but definitively determining the reasons remains elusive.
```

Comment:
The open quotes don't render right in the html, maybe switch from '' to " and then check in the rendered html.